systems/plants/collisionDetectmex.cpp

```diff
@@ -19,9 +19,6 @@ using namespace std;
  */
 
 void mexFunction( int nlhs, mxArray *plhs[],int nrhs, const mxArray *prhs[] ) {
-  //DEBUG
-  //cout << "collisionDetectmex: START" << endl;
-  //END_DEBUG
 
   if (nrhs < 1) {
     mexErrMsgIdAndTxt("Drake:collisionDetectmex:NotEnoughInputs","Usage collisionDetectmex(model_ptr)");
```

Comment:
update this to describe the (optional?) active_collision_options and allow_multiple_contacts